posts/2021/10/jupyter-community-calls.md

```diff
@@ -0,0 +1,102 @@
+| [May](https://www.youtube.com/watch?v=g9WrS4FlkeM&list=PLUrHeD2K9Cmkoamm4NjLmvXC4Y6E1o8SP&index=13)       | [June](https://www.youtube.com/watch?v=-aQFh8rjcdA&list=PLUrHeD2K9Cmkoamm4NjLmvXC4Y6E1o8SP&index=14)     | [July](https://www.youtube.com/watch?v=ebUXrmRfkM8&list=PLUrHeD2K9Cmkoamm4NjLmvXC4Y6E1o8SP&index=15)  | [August](https://www.youtube.com/watch?v=WQ43ywKDfg8&list=PLUrHeD2K9Cmkoamm4NjLmvXC4Y6E1o8SP&index=16) |
+| [September](https://www.youtube.com/watch?v=BZLM_clyHwo&list=PLUrHeD2K9Cmkoamm4NjLmvXC4Y6E1o8SP&index=17) | [October](https://www.youtube.com/watch?v=ik2hG10GlE4&list=PLUrHeD2K9Cmkoamm4NjLmvXC4Y6E1o8SP&index=18)  |
+
+## Some highlights from the past year
+
+Each event had its own charm as the attendees varied. Below we highlight some interesting outcomes:
+
+* Jupyter underwent a major version change - 2.x to 3.x - during the past year. 
+  The monthly syncs were critical in keeping the community aware of what is coming and weather the storm together. 
+  More recently, we're seeing the fruits of the [Jupyter 4.0 roadmap][40] at our meetings each month.
+* We received updates from the [JupyterHub](https://jupyter.org/hub) and [BinderHub](https://github.com/jupyterhub/binderhub) communities, 
+  which support scaling Jupyter's abilities.
+* We saw the _first_ browser based implementation of Jupyterlab called [jupyterlite](https://jupyterlite.readthedocs.io/en/latest/) [^jyve]. 
+* We explored several cool extensions such as [ipydrawio](https://github.com/deathbeds/ipydrawio), 
+  [jupyter-lsp](https://github.com/jupyter-lsp/jupyterlab-lsp), 
+  [internationalization](https://jupyterlab.readthedocs.io/en/stable/extension/internationalization.html), 
+  [janki](https://github.com/gt-coar/janki), [jupyter-videochat](https://github.com/jupyterlab-contrib/jupyter-videochat), 
+  [citations](https://github.com/krassowski/jupyterlab-citation-manager), 
+  [jupyter-h5web](https://github.com/silx-kit/jupyterlab-h5web)
+* We saw the formation of a security-focused working group and governance updates.
+
+## More community calls
+
+We'd love to see more events like this cater properly to Jupyter's global audience. 
+If that's something you care about too, please [contact us by opening an issue on our planning repository](https://github.com/Quansight-Labs/jupyter-communitycalls/issues). 
+We've treated these events as an open source project and have a lot of resources to get you started in the [technical resources section](#technical-resources).
+
+We're all nostalgic for the conference hallway track as we stay safe in a pandemic. 
+With the community calls, for one hour a month, we've been doing our best to simulate the hallway track and the communitas it can bring. 
+We hope to see you and your notebooks at an up and coming event.
+
```

Comment:
Based on @melissawm's feedback, here's a potential rephrasing of the resources to mention what we want from the future of calls. If we want to commit this, it may be worth removing the `## Technical resources` section.

```suggestion
What’s up next? For Jupyter, we are looking forward to another year of community calls with a focus on reaching out to communities we heard less from in 2021. This could be other Jupyter projects, people using Jupyter in educational settings, or someone we haven’t met yet.

At a larger scale, we hope to find even more community calls happening across the scientific Python ecosystem! This community call format is designed to be reproducible, and we have resources ready for you to get started yourself available in the [how-to documentation on our repository](https://github.com/Quansight-Labs/jupyter-communitycalls/blob/main/how-to-use-this-repo.md). 
```